test/e2e/self_hosted_test.go

```diff
@@ -74,7 +75,8 @@ func TestCreateSelfHostedClusterWithBootMember(t *testing.T) {
 	defer e.Close()
 
 	<-e.Server.ReadyNotify()
-	fmt.Println("etcdserver is ready")
+
+	t.Log("etcdserver is ready")
```

Comment:
print in test is wrong. test should generate no output if it succeeds. if you want to debug, add debugging line and disable by default.

When test fails, it should print out log. So using https://golang.org/pkg/testing/#T.Log is correct. And this is also why t.log designed this way at the first place.